src/tools/pontoon/search_filters.md

```diff
@@ -55,3 +55,20 @@ For specific projects it’s also possible to filter strings based on *tags*. Ta
 In this case, there are 10 tags defined for the project (highlighted in red). Near each tag there is a representation of the priority: like for projects, it goes from 5 stars (highest priority) to 1 star (lowest priority).
 
 Note: translation time, translation authors, and tags are not available when `All Projects` is selected as a [resource](glossary.md#resource).
+
+## Options
```

Comment:
This section should be under Search, not Filters.

Reply:
Good point. Worth noting that the search filter will be "below" at that point (next paragraph).